src/components/KSlideout/KSlideout.vue

```diff
@@ -48,6 +50,24 @@ const props = defineProps({
     type: Boolean,
     default: false,
   },
+  // controls close button alignment
+  closeBtnAlignment: {
```

Comment:
suggestion: don't abbreviate prop names
```suggestion
  closeButtonAlignment: {
```